sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -12,6 +12,13 @@
       <v-col>
         <div class="label-value-normal">
           <slot name="content">{{ summary }}</slot>
+          <slot name="content-list" v-if="summaryList">
+            <ul>
+              <li v-for="item in summaryList" :key="item">
```

Comment:
Good job working with the `v-for`!

I think that would be more appropriate using the index for the key though:

```ts
<div v-for = "(title, index) in news" :key="index">
     {{ title }}
</div>
```

Reply:
Updated